components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt

```diff
@@ -22,6 +22,16 @@ import mozilla.components.support.base.log.logger.Logger
 class GleanDebugActivity : Activity() {
     private val logger = Logger("glean/GleanDebugActivity")
 
+    companion object {
+        // This is a list of the currently accepted commands
+        const val SEND_PING_EXTRA_KEY = "sendPing"
+        const val LOG_PINGS_EXTRA_KEY = "logPings"
+        const val TAG_DEBUG_VIEW_EXTRA_KEY = "tagPings"
+
+        // Regular expression filter for debugId
+        val pingTagPattern = "[a-zA-Z0-9-]{1,20}".toRegex()
```

Comment:
nit: shouldn't this be a `const val PING_TAG_PATTERN =...` as well?

Reply:
`Error: const 'val' has 'Regex'.  Only primitives and String are allowed`

I tried that before, but the error above keeps me from declaring a const val Regex...

Reply:
Woha, I didn't know about that :)

Reply:
It would work if I left it as a string and only converted it to a Regex when I use it.

Reply:
Your call!